CMP1903_A1_2324/Die.cs

```diff
@@ -14,10 +14,13 @@ internal class Die
          */
```

Comment:
Add XML comments to each class

Reply:
I agree with Morgan, XML Comments will allow for a more in-depth read of your codebase.